src/containers/NewNote/NewNote.tsx

```diff
@@ -77,17 +95,18 @@ const NewNote: FC<RouteComponentProps> = props => {
           onChange={handleFileChange}
         />
         <label htmlFor="file">
-          <Button variant="outlined" component="span">
+          <Button variant="outlined" component="span" id="uploadFileButton">
             Upload
           </Button>
           &nbsp;&nbsp;&nbsp;&nbsp;
           {fileName ? fileName : 'Select a file'}
         </label>
         <LoaderButton
+          id="createButton"
           type="submit"
           variant="contained"
           color="primary"
-          isLoading={isLoading}
+          isLoading={creatingNote}
```

Comment:
I think the lesser variables we need to keep track in our head the better. You can call it isLoading at declaration.

Reply:
Yep will do